independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/resolver/maven/DependencyGraphLogger.java

```diff
@@ -0,0 +1,74 @@
+/**
+ *
+ */
+package io.quarkus.bootstrap.resolver.maven;
+
+import org.codehaus.plexus.util.StringUtils;
+import org.eclipse.aether.graph.DependencyNode;
+import org.eclipse.aether.graph.DependencyVisitor;
+import org.eclipse.aether.util.graph.manager.DependencyManagerUtils;
+
+public class DependencyGraphLogger
+        implements DependencyVisitor {
+
+    private String indent = "";
+
+    public boolean visitEnter(DependencyNode node) {
+        StringBuilder buffer = new StringBuilder(128);
+        buffer.append(indent);
+        org.eclipse.aether.graph.Dependency dep = node.getDependency();
+        if (dep != null) {
+            org.eclipse.aether.artifact.Artifact art = dep.getArtifact();
+
+            buffer.append(art);
+            buffer.append(':').append(dep.getScope());
```

Comment:
I can't remember if toString give the classifier and packaging / extension.

Reply:
Thanks, this class is not used at all!

Reply:
Should it be deleted?

Reply:
Sure, deleted.